### ISSUE TITLE
Forvo cloud stopped working 10 hours ago

### ISSUE THREAD

**Issue by voothi** [FreeLanguageTools/vocabsieve#146]
**Describe the bug**
Hello! 
Forvo cloud stopped working 10 hours ago. I tried version 0.11.1 and 0.12.0.
I also tried through different internet connections.

**To Reproduce**
In GUI VocabSieve 0.11.1 / 0.12.0 under Windows 11.
In mode
General / Target language = German
I connect Forvo to
Configure / Sources / Pronunciation sources / Enabled pronunciation sources / Forvo.
Checking the settings
Lemmatization policy for pronunciation = Try lemma first, otherwise original
I make Lookup words from the captured sentence.
The word is not reproduced.
The forvo/... file below is not displayed. Only local audio files from local audio libraries are displayed.

**Expected behavior**
Audio file for selected German word:
- Downloaded from cloud Forvo.
- Displayed at the bottom of the application's main screen.
- Plays automatically.

**Screenshots**
<img width="745" alt="image" src="https://github.com/FreeLanguageTools/vocabsieve/assets/9587340/09edfe12-44fc-42a9-a0b2-5fdd17ebea13">
<img width="450" alt="image" src="https://github.com/FreeLanguageTools/vocabsieve/assets/9587340/1636bc4b-f9ff-43e0-9fca-8e255aec7260">

**Logs**
-

**Desktop (please complete the following information):**
- OS: Windows 11.
- Vocabsieve version (if nightly, must be latest): VocabSieve 0.11.1 / 0.12.0.

**Additional context**
Telegram thread:
Hello! Forvo stopped working 10 hours ago. I tried version 0.11.1 and 0.12.0.

**Comment by 1over137** [comment]
I assume they stepped up their anti scraping measures with Cloudflare, as reported elsewhere with similar projects:

https://github.com/Rascalov/Anki-Simple-Forvo-Audio/issues/31

https://github.com/Rascalov/Anki-Simple-Forvo-Audio/issues/29

In that case there is nothing I can do. Stick to downloaded stuff, probably. If this persists I'll remove the implementation. Maybe I'll look into fetching audio from wiktionary.

**Comment by Rascalov** [comment]
>Сейчас зашел в Anki, и все заработало, на сайте Forvo не заходил, но если появится данная проблема еще раз, сделаю, как вы сказали. Еще раз спасибо за этот прекрасный аддон

No problem! But I assume you wanted to send this reply in [your own created issue](https://github.com/Rascalov/Anki-Simple-Forvo-Audio/issues/31) 😅

**Comment by 1over137** [comment]
From what I see, they are still under Cloudflare reverse proxy. The DNS points to a cloudflare IP: http://104.20.253.20/. I assume they are testing this. Most likely they'll just block it again permanently, and then there won't be good solutions other than a full browser scraping script.

**Comment by Rascalov** [comment]
yeah, there are people ([freemdict](https://forum.freemdict.com/)) that have were already [posting full site scrapes of audios](https://cloud.freemdict.com/index.php/s/pgKcDcbSDTCzXCs). Can't say how recent these are though.
